pkgs/development/python-modules/json-schema-for-humans/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, fetchurl
+, pbr, click, dataclasses-json, htmlmin, jinja2, markdown2, pygments, pytz, pyyaml, requests, pytest, beautifulsoup4
+}:
+
+buildPythonPackage rec {
+  pname = "json-schema-for-humans";
+  version = "0.27.0";
+  src = fetchFromGitHub {
+    owner = "coveooss";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1r40i192z6aasil5vsgcgp5yvx392dhhqnfc2qxbxvpja6l3p6p2";
+  };
+  patches = [ (fetchurl {
+    url = "https://github.com/coveooss/json-schema-for-humans/commit/1fe2e2391da5a796204fd1889e4a11a53f83f7c9.patch";
+    sha256 = "0kpydpddlg0rib9snl8albhbrrs6d3ds292gpgpg7bdpqrwamdib";
+  }) (fetchurl {
+    url = "https://github.com/coveooss/json-schema-for-humans/pull/66.patch";
```

Comment:
Please do not fetch PRs but specific commits. If the PR changes the hash changes which we try to avoid.